frontend/app-config/webpack.config.js

```diff
@@ -204,6 +206,26 @@ module.exports = function (env) {
                 // Add errors to webpack instead of warnings
                 failOnError: true
             }),
+            new plugins.CopyPlugin({
+                patterns: [
+                  { from: './node_modules/simplemde/dist', to: 'dependencies/simplemde' },
+
+                  { from: './node_modules/tinymce/tinymce.min.js', to: 'dependencies/tinymce/tinymce.min.js' },
```

Comment:
I think I would use wildcard here and copy just everything.

Reply:
Are you sure? tinymce includes quite a lot of plugins and garbage files that are not needed. Will result in quite a lot of logs like so:

```
                                                                Asset        Size  Chunks             Chunk Names
                                               dependencies/ace/ace.js     371 KiB          [emitted]  
```

Reply:
Makes sense.